### ISSUE TITLE
Upload file from cloud storage accessibility audit 

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#3982]
### Descriptive summary

QA Testing for Hyrax
Hyrax Version: 3.0.0.pre.rc1
Hyrax Branch: master
Hyrax Revision: 48ed925
Nurax Version
ENV: production
SHA: 40b47b0d71af1b958e016ebe49212c1740683133
Branch: 2019-09-12-10-00
Updated: 12 Sep 2019 15:00:37

In testing for the Hyrax 3.0 release, the following points are actions that were discovered during a FireFox tab test and axe accessibility audit. Note, only critical issues are being reported at this time.

**id attribute value must be unique**
![Screen Shot 2019-09-12 at 5 30 58 PM](https://user-images.githubusercontent.com/9621070/64829917-25049480-d583-11e9-93c5-4851becc05ce.png)

**ARIA role must be appropriate for the element**
![Screen Shot 2019-09-12 at 5 31 48 PM](https://user-images.githubusercontent.com/9621070/64829940-39e12800-d583-11e9-8573-19cac1c76de5.png)

**Heading levels should only increase by one**
![Screen Shot 2019-09-12 at 5 32 59 PM](https://user-images.githubusercontent.com/9621070/64829981-66953f80-d583-11e9-931c-f3edf7ae3e68.png)

**Aside must not be contained in another landmark**
![Screen Shot 2019-09-12 at 5 34 05 PM](https://user-images.githubusercontent.com/9621070/64830049-8e84a300-d583-11e9-86e8-6e730e17fee2.png)

![Screen Shot 2019-09-12 at 5 34 46 PM](https://user-images.githubusercontent.com/9621070/64830064-a3f9cd00-d583-11e9-981d-c02d2ab4d778.png)

**Main landmark must not be contained in another landmark**
![Screen Shot 2019-09-12 at 5 35 24 PM](https://user-images.githubusercontent.com/9621070/64830080-ba078d80-d583-11e9-895c-6135e01cce79.png)

**Document must have one main landmark**
![Screen Shot 2019-09-12 at 5 36 04 PM](https://user-images.githubusercontent.com/9621070/64830101-d0ade480-d583-11e9-8fa6-6ab5d1ecbe1f.png)

**Ensures landmarks are unique**
![Screen Shot 2019-09-12 at 5 36 43 PM](https://user-images.githubusercontent.com/9621070/64830128-e91dff00-d583-11e9-8c6e-6fedcdfd726b.png)

![Screen Shot 2019-09-12 at 5 37 26 PM](https://user-images.githubusercontent.com/9621070/64830161-04890a00-d584-11e9-8e76-60b1b581061e.png)

**Radio inputs with the same name attribute value must be part of a group**

![Screen Shot 2019-09-12 at 5 56 14 PM](https://user-images.githubusercontent.com/9621070/64830678-a3166a80-d586-11e9-920d-80993df5854b.png)

### Steps to reproduce the axe accessibility audit

1. Install the Axe plugin or addon from Deque (https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/ is the firefox version).
2. Open https://nurax-dev.curationexperts.com/concern/generic_works/th83kz52k/edit?locale=en#files and click Add cloud files 
3. Open the inspector (right-click, and select "Inspect element")
4. Click on the 'Axe' tab
5. Click 'Analyze'

### Video of Tab Test:

In Firefox and Safari, Connect to Box and Submit buttons are not accessible via tab test.

Chrome is working properly. 

https://cl.ly/21b209864476/download/Screen%252520Recording%2525202019-09-12%252520at%25252005.27%252520PM.mov

**Comment by rjkati** [comment]
Tab test passes in Hyrax 4.0.0 on nurax-dev. I could not get "Add cloud files" box to display locally in Hyrax 3.5.0.

axe plugin is unavailable for Firefox, but reported issues are not present in Hyrax 4.0.0 using WAVE for Firefox.
